README.md

```diff
@@ -39,9 +39,11 @@ well. For more, see the [Code of Conduct](https://swift.org/community/#code-of-c
 
 ## Getting Started
 
-These instructions give the most direct path to a working Swift
-development environment. Options for doing things differently are
-discussed below.
+These instructions give the most direct path to a working Swiftdevelopment
+environment. To build from source you will need 2gb of disk space for the
+source code and over 20gb of disk space for the build artifacts. The first
+build will take multiple hours, but incremental builds will finish much faster.
```

Comment:
Maybe it just *feels* like hours sometimes. I'm going to run a few builds out of my own curiosity, but downgraded this to a "can".